### DISCUSSION TITLE
Is the generate polygon distance snap in the editor supposed to be affected by Slider Velocity?

### DISCUSSION THREAD

**Discussion by Alib234** [ppy/osu#29825] [Q&A] [answered]
Because currently it is, for example if SV is 3, and the distance snap of generate polygon is set to 1 the actual distance snap between the generated circles is going to be 3 not 1

**Comment by bdach** [comment, marked as answer]
You mean the base beatmap velocity? Yeah it is. I'm reasonably sure I cross-checked against stable (and I did so again just now) and in stable that is what happens.

**Reply by Alib234** [reply]
I meant inherited points(green) in stable or just slider velocity on sliders in lazer ~~tbh I never used this feature in stable~~ but after checking stable it seems like it also confusingly gets affected by slider velocity changes in inherited points in stable as well
